### PR TITLE
Popup: Use .done() instead of .then() for prerequisites

### LINKED ISSUE #8044: Transitions: Tests fail with core from git
Something must've changed in core, because our transitions tests are failing with core from git.

### PR DESCRIPTION
**Pull request by gabrielschulhof**
Fixes gh-8044

### CODE REVIEW

**Comment by arschmitz** [comment]
Ref jquery/jquery#1996

File: tests/integration/select/index.html [hunk 1]
```diff
@@ -27,7 +27,6 @@
 
 	<link rel="stylesheet"  href="../../../css/themes/default/jquery.mobile.css"/>
 	<link rel="stylesheet" href="../../../external/qunit/qunit.css"/>
-	<link rel="stylesheet" href="../../jqm-tests.css"/>
```
Comment (arschmitz) [inline tests/integration/select/index.html]: Why is this being removed?

Comment (gabrielschulhof) [inline tests/integration/select/index.html]: D'oh! Sorry! I'll put it back.

**Comment by arschmitz** [comment]
:+1:

**Comment by gabrielschulhof** [comment]
Closed in d34c86ae14d4ea603357be5e326de1fb8c31dbf9

**Pull request closed without merging**